### ISSUE TITLE
Refactor website to be more mobile friendly and update items to be more design sensible

### ISSUE THREAD

**Issue by MoonWolf125** [MSUM-ACM/MSUM-ACM.github.io#25]
While the website naturally handles format adjustment between various screen sizes/densities it can be enhanced to be more in line with Material Design.

https://material.io/design/

**Comment by MoonWolf125** [comment]
We can also use [Figma](Figma.com) to possibly create a design that we can follow

**Comment by alunduil** [comment]
Is the focus of this issue to make the website layout reactive?

**Comment by MoonWolf125** [comment]
More so to clean up the reactivity. Jekyll is reactive out of the box, but it hasn't been updated to a modern design.

**Comment by MoonWolf125** [comment]
Created a Figma Design for the app: https://www.figma.com/file/vxTRZAVaQ3qFwNxE1KkRIL/Design

Link is view only. Free teams are very restrictive. Link will be included in the next update once the design is at least up to date with the current site.

**Comment by MoonWolf125** [comment]
If @rmk8512 @Toi-Sh @Sleekice and @ankurlamichhane90 want to take a second and take a look at the Figma link above that would be wonderful! Please leave your feedback. Hoping that maybe we can bring the website a new face this year.

**Comment by rmk8512** [comment]
I like the design and mobility! It'll be great to have an app for ACM!

**Comment by Toi-Sh** [comment]
It's pretty awesome!

**Comment by MoonWolf125** [comment]
This is a design for the website.

**Comment by Toi-Sh** [comment]
My bad, I was looking at material design! But I saw the new look for the site, I like the red background.

**Comment by rmk8512** [comment]
Oh sorry, I was looking at another page. I like the new update, it's simple and clean! love the little MSUM penguin on top of the website too

**Comment by Sleekice** [comment]
Correct me if saw wrong, but it seems to be in it's initial stage of development and i think the background links it to MSUM which is a great idea as that's our school color. Also, nothing wrong with using 'home page' as the main page. But, that's just me.

**Comment by MoonWolf125** [comment]
Yeah, designs are in the initial phase since I'm not on top of my web programming and design. I've been referencing the visual guidelines as released by the school's design and marketing department. The colors are the same colors that the school uses for all documents and content. I've outlined the colors in the Design as outlined in the Visual Guidelines. I'm fairly sure we'll use the home page as the main page. That's how it's been since the beginning of the previous design.

**Comment by MoonWolf125** [comment]
Updated the designs again @ankurlamichhane90 @Sleekice @Toi-Sh @rmk8512 

I wasn't really feeling the white toolbar with the black text. It also didn't necessarily match the Material Design site's toolbar. I made the toolbar the same red, double checked that they all had drop shadows, changed all the text to white, and gave the penguin a white circular background so he doesn't blend in with everything.
